include/filehandler/pemhandler.h

```diff
@@ -5,16 +5,15 @@
 #include <vector>
 
 #include "ifilehandler.h"
-#include "wrappers/pem_input_wrapper.h"
 #include "account/address.h"
 #include "internal/internal.h"
 
 namespace ih
 {
-class PemInputHandler : public IFileHandler
+class PemFileReader : public IFileHandler
```

Comment:
👍 _reader_ makes much more sense. 

Even _IFileHandler_ can be renamed to _IFileReader_, or even _IFile_, I think. `Handler` is a bit imprecise, ambiguous (especially because of the existing term _file handles_) and - if it's sole purpose is to mark a type as `interface` - redundant, since you have the `I` prefix anyway.

Reply:
I would go for IFile, as we will probably add also "write" method(s) to be used when we generate a pem file for example.

Reply:
Good point ! Will change its name to _IFile_ !

Reply:
Also renamed pemhandler -> pemreader